Sources/Sentry/Public/SentryOptions.h

```diff
@@ -287,6 +287,7 @@ NS_SWIFT_NAME(Options)
  */
 @property (nonatomic, assign) BOOL enableAutoPerformanceTracing;
 
+#if !SDK_V9
```

Comment:
`h`: Not sure if I understand this PR's intention. Do you want to make `enablePerformanceV2` be enabled by default or do you want to remove the options? If the latter one why would we want to remove the option?

Reply:
We should remove the option and enable everything by default that depends on performanceV2 in the next major. `performanceV2`. Currently that's only 
> The app start duration will now finish when the first frame is drawn instead of when the OS posts the

We didn't change this in a minor, because it impacts the app start duration, and it could cause alerts. 

I thought we would make more changes with this feature flag, but then our performance work improvements got deprioritized, and we never made further improvements. This option adds complexity, and we should ditch it.

Reply:
I understand, in that case yes let's change this for V9.

We need to start a v8 -> v9 migration guide!